checker/src/main/java/org/checkerframework/checker/nullness/messages.properties

```diff
@@ -16,3 +16,5 @@ new.class.type.invalid=the annotations %s do not need be applied in object creat
 toArray.nullable.elements.not.newarray=call of toArray on collection of non-null elements yields an array of possibly-null elements; omit the argument to toArray or make it an explicit array constructor
 toArray.nullable.elements.mismatched.size=call of toArray on collection of non-null elements yields an array of possibly-null elements; cannot determine that the argument array has the same size as the receiver collection
 clear.system.property=call might clear a predefined system property; pass -Alint=permitClearProperty to permit it
+instanceof.nullable.error=instanceof is only true for a non-null expression
```

Comment:
".error" makes this key longer without making it more informative.  I suggest shortening the key.